File: app/src/main/java/com/zillennium/utswap/screens/setting/loginPasswordScreen/LoginPasswordActivity.java
package com.zillennium.utswap.screens.setting.loginPasswordScreen;

import android.os.Bundle;
import android.widget.ImageView;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.zillennium.utswap.bases.BasePassedActivity;
import com.zillennium.utswap.databinding.ActivitySettingLoginPasswordBinding;

public class LoginPasswordActivity extends AppCompatActivity {

    ActivitySettingLoginPasswordBinding binding;

    @Nullable
    protected void onCreate(Bundle savedInstanceState){
        super.onCreate(savedInstanceState);

        binding = ActivitySettingLoginPasswordBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());



        // Set Passed Back
        ImageView ivBack = binding.ivBack;
        BasePassedActivity.onPassedBack(LoginPasswordActivity.this, ivBack);
    }

}

File: app/src/main/java/com/zillennium/utswap/screens/setting/loginPasswordScreen/LoginPasswordActivity.kt
package com.zillennium.utswap.screens.setting.loginPasswordScreen

import com.zillennium.utswap.bases.mvp.BaseMvpActivity
import com.zillennium.utswap.R
import com.zillennium.utswap.databinding.ActivitySettingLoginPasswordBinding
import java.io.IOException

class LoginPasswordActivity :
    BaseMvpActivity<LoginPasswordView.View, LoginPasswordView.Presenter, ActivitySettingLoginPasswordBinding>(),
    LoginPasswordView.View {

    override var mPresenter: LoginPasswordView.Presenter = LoginPasswordPresenter()
    override val layoutResource: Int = R.layout.activity_setting_login_password

    override fun initView() {
        super.initView()
        try {
            binding.apply {
                ivBack.setOnClickListener {
                    onBackPressed()
                }
            }
        } catch (error: Exception) {
            // Must be safe
        }
    }
}